pkg/pki/openshift.go

```diff
@@ -32,22 +32,32 @@ func GeneratePKI(params *api.ClusterParams, outputDir string) error {
 		return errors.Wrapf(err, "failed to parse service CIDR: %q", params.ServiceCIDR)
 	}
 	kubeIP := firstIP(serviceIPNet)
+	apiServerHostNames := []string{
+		"kubernetes",
+		"kubernetes.default.svc",
+		"kubernetes.default.svc.cluster.local",
+		"kube-apiserver",
+		fmt.Sprintf("kube-apiserver.%s.svc", params.Namespace),
+		fmt.Sprintf("kube-apiserver.%s.svc.cluster.local", params.Namespace),
+	}
+	apiServerIPs := []string{
+		kubeIP.String(),
+		params.ExternalAPIIPAddress,
```

Comment:
don't you want this removed, and only added when line 47 falls through that condition?

Reply:
It's different... this is the ExternalAPI**IP**Address, which I always want in the IPs list

Reply:
ah yes, i see it now